gulp-tasks/documentation.js

```diff
@@ -31,33 +31,40 @@ navigation_weight: 0
 
 const AUTO_GENERATED = '<!-- DO NOT EDIT. This page is autogenerated. -->';
 
+const generateHandlebarsData = (projectPath) => {
+  const projectMetadata = require(`${projectPath}/package.json`);
+
+  const handleBarsData = {
+    name: projectMetadata.name,
+    description: projectMetadata.description,
+    background: projectMetadata.background,
+  };
+
+  try {
+    const demoStats = fs.statSync(path.join(projectPath, 'demo'));
+    handleBarsData.hasDemo = demoStats.isDirectory();
+  } catch(err) {
+    handleBarsData.hasDemo = false;
```

Comment:
Is there a reason this is in a try/catch block vs `.hasDemo = !!demoStats.isDirectory()`?